benches/pk_encryption/ring_lpr.rs

```diff
@@ -0,0 +1,288 @@
178 82392189 80348703 13091429 16537571 28523203 56415594 52318523 38855189 70158090 91770055 46586539 74993580 22862135 40069112 67171718 23976047 26808811 40719731 63077026 49306550 84975623 67619507 92536260 49121230 34091697 71911227 16299605 78001062 2594634 30953023 80123432 62944594 55653346 61441215 73568042 24980986 14828884 55089628 55050668 70990560 46838871 52907405 23907531 6546518 1242457 67168672 15850320 23345906 6128635 23752062 38601718 67855498 50403576 81899196 70833756 26170206 92125762 72617678 8805563 31698131 55049905 87503078 31711331 13929493 67428740 26782899 80632642 55085086 71330359 5510841 39549117 47668062 8054414 60133144 3977509 49988560 6132017 8942257 72859536 24292110 60928824 26389413 47155245 17470234 61399580 71361792 46240962 32517837 48720795 83047250 24844689 9173856 28049592 33071141 31908987 5636031 26350980 78811596 23582264 10061719 37490140 39997002 29151082 52761302 90967695 42822883 42204422 22482987 46674812 80782632 15510611 20017335 74747531 53116712 25940019 92818987 81035431 63119139 422440 68878886 75333410 77293603 41141547 83203318 21361070 92082423 73367510 39897286 26123207 13043331 52139348 82520994 78340740 53413734 88355538 84396174 82247860 11585366 8252961 2520142 56029962 19974849 78619484 60618225 73393597 67546165 21004921 53250858 12347361 42132026 21961346 10433677 5375241 73074480 76302602 83742758 55344059 56106028 45437058 39839534 33685238 56076577 88223410 11258415 87404750 13961165 40212667 8791760 70167954 52066968 4421881 23391756 50794215 25257756 84755661 70591604 17354866 47908906 33607995 69692057 31561454 84343526 82248718 56790341 74426028 85006096 33081759 49469639 76160135 24050718 10896378 56526566 29592005 15032920 16754061 36882444 13577453 74044319 4011174 65932557 59146756 80987745 45761964 10610454 13397730 20173285 80545807 7611338 16311320 40059461 10299710 41349998 48131787 2158641 90174574 18213624 83580890 12621310 83730716 17825996 60677807 92469077 7148155 55663541 9526476 23884810 7238618 68151793 13029971 87339755 82475413 3338308 44367833 50430781 31875410 63806931 13040338 12455459 12121072 57953591 88379188 57743291 22865104 50978503 88687482 36725608 70561052 33178061 62109489 85871352 91814121 29802978 57020869 43073503 62448381 91819557 90466761 54829533 15603461 65828354 61772468 90807274 37382884 19249379 9517877 20646188 3631235 39531847 89461002 83153463 75646752 19569200 70246055 9332258 37372925 81136283 86719383 58647744 8512334 39106336 84039335 11884100 63744177 64274951 62019805 49019697 71879755 90109919 62758919 54586613 47826418 79038398 13142583 65097569 79805514 79843767 25903241 40455284 79159820 41203717 33496537 55375236 26523661 89778467 83091139 16161544 78596363 82706865 59426806 26590827 29540228 76897672 524171 19320584 63132643 86168164 31732727 70207474 46440791 39246509 47879651 76294317 53278850 88908292 35326020 17646244 33466197 47619936 76476577 79862787 79241752 10741699 79370927 16548017 39413766 16813868\"}";
```

Comment:
here you include it as a constant, because it is shorter?
Seems a little inconsistent compared to how you did it in the general encryption case